app/antedecorators/accesscontrol_wasm_dependency.go

```diff
@@ -54,7 +54,7 @@ func (ad ACLWasmDependencyDecorator) SenderMatchesContractAdmin(ctx sdk.Context,
 	return contractInfo.Admin == msg.FromAddress, nil
 }
 
-func (ad ACLWasmDependencyDecorator) AnteDeps(txDeps []sdkacltypes.AccessOperation, tx sdk.Tx, next sdk.AnteDepGenerator) (newTxDeps []sdkacltypes.AccessOperation, err error) {
+func (ad ACLWasmDependencyDecorator) AnteDeps(txDeps []sdkacltypes.AccessOperation, tx sdk.Tx, txIndex int, next sdk.AnteDepGenerator) (newTxDeps []sdkacltypes.AccessOperation, err error) {
```

Comment:
Why are these called `AnteDeps`?

Reply:
to be similarly named to `AnteHandler` which handles the stuff prior to running messages, `AnteDeps` is meant o represent `AnteDependencyGenerator` to generate dependencies prior to message dependencies